WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -374,11 +374,11 @@ private async Task HandleCoinJoinFinalizationAsync(CoinJoinTracker finishedCoinJ
 			{
 				CoinRefrigerator.Freeze(result.RegisteredCoins);
 				MarkDestinationsUsed(result.RegisteredOutputs);
-				wallet.LogInfo($"{nameof(CoinJoinClient)} finished!");
+				wallet.LogInfo($"{nameof(CoinJoinClient)} finished. Coinjoin transaction was broadcast.");
```

Comment:
nit:
```suggestion
				wallet.LogInfo($"{nameof(CoinJoinClient)} finished. Coinjoin transaction was broadcasted.");
```

Reply:
https://english.stackexchange.com/questions/33207/broadcast-or-broadcasted

https://en.wiktionary.org/wiki/broadcasted#:~:text=The%20past%20of%20broadcast%20is,but%20also%20as%20past%20participle.